docs/spectrum/index.rst

```diff
@@ -29,7 +29,47 @@ Getting Started
 
 TODO
 
-Hello World
+Energy handling in Gammapy
+==========================
+
+.. warning::
```

Comment:
Remove this warning. It won't be more experimental than the rest of Gammapy when merged.